### ISSUE TITLE
but is it ACTUALLY safe

### ISSUE THREAD

**Issue by LeBobrze** [7OU/rushercrack#20]
there are so many mixed reviews people saying its a joke and people saying it works does it work or not

**Comment by yeetus-maximus** [comment]
yes it is just follow the instructions. on first run it took me about 8 days to crack

**Comment by amoraschi** [comment]
Yes it is, because it doesn't do anything, it just prints random things to the console

**Comment by Alx010** [comment]
Yes it's safe it just only executes random mathematical operations and nothing more

**Comment by hydraontopp** [comment]
yep totally, he can totally rat a js file, just compile it urself

**Comment by cablesalty** [comment]
It's safe, but at the same time, completely fake.
